src/pytorch_lightning/accelerators/accelerator.py

```diff
@@ -31,54 +29,9 @@ class Accelerator(ABC):
     - HPU
     """
 
-    def setup_environment(self, root_device: torch.device) -> None:
-        """Setup any processes or distributed connections.
-
-        This is called before the LightningModule/DataModule setup hook which allows the user to access the accelerator
-        environment before setup is complete.
-        """
-
     def setup(self, trainer: "pl.Trainer") -> None:
         """Setup plugins for the trainer fit and creates optimizers.
 
         Args:
             trainer: the trainer instance
         """
```

Comment:
This trainer-specific method is currently the only difference between the Accelerator base class in PL vs Lite. In the future, we will be able to refactor the usages to merge the base classes.